contracts/0.8.9/WithdrawalQueue.sol

```diff
@@ -209,7 +209,21 @@ abstract contract WithdrawalQueue is AccessControlEnumerable, PausableUntil, Wit
         uint256 hint;
     }
 
-    /// @notice Claim withdrawals batch once finalized (claimable).
+    /// @notice Return batch of claimable eth amounts that is locked for each request
```

Comment:
should note that requests must be in finalized state

Reply:
They should not, actually. It will return 0 as we have no claimable eth in non-final requests.

Reply:
got your point, yeah